INA219.cpp

```diff
@@ -149,9 +156,16 @@ void INA219::configure(  t_range range,  t_gain gain,  t_adc  bus_adc,  t_adc sh
 }
 
 #define INA_RESET        0xFFFF    // send to CONFIG_R to reset unit
-void INA219::reset(){
-  write16(CONFIG_R, INA_RESET);
-  _delay_ms(5);
+//RMW Returns false if unable to reset (likely due to device not being functional/installed)
+bool INA219::reset(){
+  uint8_t ret = write16(CONFIG_R, INA_RESET);
+  if (ret) {
+    Serial.print("INA219::reset: write16 error: ");
+    Serial.println(ret);
+    return false;
+  }
+//RMW  _delay_ms(5);
```

Comment:
Why remove this delay ?

Reply:
See above comment about FreeRTOS and vTaskDelay/delay

Reply:
Why is this delay using _delay_ms and the other using delay() ? 
And more importantly, possibly, why is _delay_ms() redefined in ESP32 as delayMicroseconds() rather than milliseconds? I think there's a bug in this somewhere.
Can we agree to standardize on delay() so that everthing is in milliseconds and there's only one delay function?
And if so, does this make it so that there's no need to redefine a delay function in the ESP32 case? It seems there might be some 'history' here that I don't have insight into.